lesson22/js/problem7.js

```diff
@@ -0,0 +1,9 @@
+let capacityOfFleshInGigabite = +prompt(
```

Comment:
```suggestion
const capacityOfFleshInGigabite = +prompt(
```
Використовуй `const`, поки не потрібно буде її переназначити.